packages/components/app/styles/components/tabs.scss

```diff
@@ -86,3 +81,25 @@ $indicator-height: 3px;
 .hds-tabs__tab-badge {
   margin-left: 6px;
 }
+
+.hds-tabs__tablist-wrapper { position: relative; }
+
+.hds-tabs__tab-indicator {
+  position: absolute;
+  right: 0;
+  // adjust position to make indicator overlap tabs bottom border:
+  bottom: floor(calc($indicator-height / 2));
```

Comment:
[issue] checking how it's designed in Figma, I think this value should be 3px (`$indicator-height`)
```suggestion
  bottom: $indicator-height;
```
<img width="1349" alt="screenshot_1852" src="https://user-images.githubusercontent.com/686239/194056438-49025857-93f2-4fc8-b9d0-8015c12a6f5f.png">

Reply:
I had deleted styles positioning the tabs when I was refactoring. I added them back so that the focus outline should now sit right above the indicator underline again. (One complicating factor is that the indicator needs to overlap the bottom gray border.)